gocourse02/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import "fmt"
+
+// Struct that represents an Animal
+type Animal struct {
+	name      string // Name of the animal
+	species   string // Species of the animal
+	inTheCage bool   // Flag indicating whether the animal is in a cage
```

Comment:
Неймінг - завжди суб'єктивна річ. Але я б перейменував так:
```suggestion
	isCaged bool   // Flag indicating whether the animal is in a cage
```

Зазвичай, але не завжди, `bool` змінні починаються з `is/if` префіксів.